src/pages/pos-pointOfSales/index.js

```diff
@@ -0,0 +1,141 @@
+import { useContext } from 'react'
+import toast from 'react-hot-toast'
+import Table from 'src/components/Shared/Table'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { useWindow } from 'src/windows'
+import { useResourceQuery } from 'src/hooks/resource'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { Fixed } from 'src/components/Shared/Layouts/Fixed'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import { ControlContext } from 'src/providers/ControlContext'
+import { PointofSaleRepository } from 'src/repositories/PointofSaleRepository'
+
+import GridToolbar from 'src/components/Shared/GridToolbar'
+import PointOfSalesWindow from './windows/PointOfSalesWindow'
+
+const PointOfSales = () => {
+  const { getRequest, postRequest } = useContext(RequestsContext)
+  const { platformLabels } = useContext(ControlContext)
+  const { stack } = useWindow()
+
+  async function fetchGridData(options = {}) {
+    const { _startAt = 0, _pageSize = 50 } = options
+
+    try {
```

Comment:
are u sure need try and catch here

Reply:
its better , if there is an error while fetching